src/ledger/backend/connection.spec.ts

```diff
@@ -28,6 +34,24 @@ describe("ErdstallConnection", () => {
 	let contract: Erdstall;
 	let conn: LedgerWriter;
 
+	const depositStagesForAssets = (assets: Assets): number => {
```

Comment:
In the spirit or renaming the generic `Stages` to the more specific term `Transactions`, this should be renamed to something like
```suggestion
	const numDepositTxs = (assets: Assets): number => {
```